src/menelaus/data_drift/hdddm.py

```diff
@@ -62,44 +62,35 @@ class HDDDM(HistogramDensityMethod):
             * The Epsilon values for each feature are stored, for each set of
               reference and test batches. For each feature, these values
               represent the difference in Hellinger distances within the test
-              and reference batch at time t, to the Hellinger distances within
-              the test and reference batch at time t-1. These can be acccessed
+              and reference batch at time *t*, to the Hellinger distances within
+              the test and reference batch at time *t-1*. These can be acccessed
               with each update call using the ``self.feature_epsilons`` variable.
               They also can be accessed when drift occurs using the
               ``self.feature_info`` dictionary.
 
         * The original algorithm cannot detect drift until it is updated with
-          the third 3rd test batch after initial setup or 3rd test batch after
-          each drift detection. When HDDDM is updated with the first test batch
-          (T=1), the distance is calculated between test and reference batch.
-          When updated with the second test batch (T=2), HDDDM calculates the
-          distance and Epsilon, the difference in distances. When HDDDM is
-          updated with the third test batch (T=3), the distance and Epsilon are
-          calculated. The adaptive threshold Beta is computed using Epsilon
-          values from lambda -> T-1 (lambda is either T=0 or the test batch
-          number on which drift was last detected). Beta needs at least one
-          prior value of Epsilon to be calculated. Now that Beta can be
-          calculated, HDDDM can detect drift on the 3rd test batch. In order to
-          allow for more immediate detection of drift, we added the following
-          options, specified through the parameter ``detect_batch``:
+          the third test batch after either a) initilization or b) reset upon
+          drift, because the threshold for drift detection is defined from the
+          *difference* Epsilon. To have at sufficient values to define this
```

Comment:
small typo - "to have sufficient values" instead of "to have at sufficient values"